generator/namespaces.py

```diff
@@ -350,23 +364,33 @@ def namespaces(
 
     namespaces = {}
     for app in glean_apps:
-        looker_views = _get_looker_views(app, db_views)
-        explores = _get_explores(looker_views)
-        views_as_dict = {view.name: view.as_dict() for view in looker_views}
-
-        namespaces[app["name"]] = {
-            "owners": app["owners"],
-            "pretty_name": app["pretty_name"],
-            "views": views_as_dict,
-            "explores": explores,
-            "glean_app": True,
-        }
+        if app["name"] not in ignore:
+            looker_views = _get_looker_views(app, db_views)
+            explores = _get_explores(looker_views)
+            views_as_dict = {view.name: view.as_dict() for view in looker_views}
+
+            namespaces[app["name"]] = {
+                "owners": app["owners"],
+                "pretty_name": app["pretty_name"],
+                "views": views_as_dict,
+                "explores": explores,
+                "glean_app": True,
+            }
 
     if custom_namespaces is not None:
         custom_namespaces = yaml.safe_load(custom_namespaces.read()) or {}
+        # remove namespaces that should be ignored
+        for ignored_namespace in ignore:
+            del custom_namespaces[ignored_namespace]
 
         # generating operational monitoring namespace, if available
         if "operational_monitoring" in custom_namespaces:
+            if use_cloud_function:
+                raise Exception(
+                    "Cannot generate OpMon using dry run Cloud Function"
+                    + "`export USE_CLOUD_FUNCTION=False` to use user GCP credentials"
+                )
+
```

Comment:
operational monitoring is actually querying some tables to get information that is needed for generating artifacts. With the dry run function these artifacts cannot be generated. 
This is something to be addressed in a future PR.